docs/development/security.md

```diff
@@ -27,7 +27,8 @@ We are delighted when solutions are offered together with the initial report. Pl
 Please subscribe to our admin mailing list (ilias-admins@lists.ilias.de) to get notifications about security updates, updates in general and announcements for ILIAS server administrators.
 
 ## Contributors
-* Fabian Sesterhenn, TH Köln, Cologne, Germany
-* Nadimo Staszak, University of Cologne, Cologne, Germany
 * Robin Baumgartner, studer + raimann ag, Burgdorf, Switzerland
```

Comment:
@fsesterh thank you for keeping the information up to date! Could you please also update our company name while you're at it? It's "sr solutions ag" now (still in Burgdorf, Switzerland).

Reply:
Done.